build.gradle

```diff
@@ -537,6 +537,9 @@ project('test:integration') {
         jvmArgs = ["-server", "-Xmx2g", "-XX:+HeapDumpOnOutOfMemoryError",
                    "-Dlog.dir=PRAVEGA_APP_HOME/logs",
                    "-Dlog.name=selftest"]
+        beforeTest { descriptor ->
```

Comment:
Why did you do this just for integration tests? We should have consistency across our tests.